src/main/java/com/flowingcode/addons/applayout/AppLayout.java

```diff
@@ -19,101 +19,110 @@
  */
 package com.flowingcode.addons.applayout;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Tag;
 import com.vaadin.flow.component.dependency.CssImport;
 import com.vaadin.flow.component.dependency.JsModule;
 import com.vaadin.flow.component.dependency.NpmPackage;
 import com.vaadin.flow.component.html.Div;
 import com.vaadin.flow.component.html.Image;
 import com.vaadin.flow.server.InitialPageSettings;
 import com.vaadin.flow.server.PageConfigurator;
-import java.util.ArrayList;
-import java.util.Arrays;
 
 /**
  * Component that renders the div that contains the entire layout.
  *
  * @author mlopez
  */
 @SuppressWarnings("serial")
-@NpmPackage(value = "@polymer/app-layout", version = AppLayout.NPM_VERSION)
-@NpmPackage(value = "@polymer/font-roboto", version = "^3.0")
-@JsModule("@polymer/app-layout/app-scroll-effects/app-scroll-effects.js")
-@JsModule("@polymer/font-roboto/roboto.js")
-@CssImport("./styles/applayout-styles.css")
+@Tag("fc-applayout")
+@JsModule("@flowingcode/fc-applayout/fc-applayout.js")
+@NpmPackage(value = "@flowingcode/fc-applayout", version = "~0.9.3")

```

Comment:
Use a strict version `0.9.3` instead of a version range, since the targeted version is experimental.

Reply:
Done ✔️